osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -301,8 +303,13 @@ private void removeGroup(BeatmapGroup group)
             foreach (var p in group.BeatmapPanels)
                 panels.Remove(p);
 
-            scrollableContent.Remove(group.Header);
-            scrollableContent.RemoveRange(group.BeatmapPanels);
+            // Must check if panels aren't already removed because they're offscreen

```

Comment:
I submitted this pull request way back on June 18th (an exact month from now). At the time remove was like this:

```
if (!internalChildren.Remove(drawable))
    throw new InvalidOperationException($@"Cannot remove a drawable ({drawable}) which is not a child of this ({this}), but {drawable.Parent}.");
```

So this can probably be removed now.